### PR TITLE
Retropie desktop app

### PR DESCRIPTION
**Pull request by mobilegmYT**
@Botspot

### CODE REVIEW

**Review by fabianmendes — CHANGES_REQUESTED** [review]
I think this is not correct, or it actually is?

File: apps/Retropie/install [hunk 1]
```diff
@@ -20,3 +20,15 @@ sudo ./retropie_setup.sh
...
+echo '[Desktop Entry]
+Type=Application
+Name=RetroPie
+GenericName=RetroPie
+Comment=EmulationStation
+Icon=/home/pi/.icons/retropie.png
+Exec=sh -c "sudo chvt 1 && sudo ttyecho -n /dev/tty1 'emulationstation; sudo chvt 7'"
+Categories=Emulator;Game;
+Keywords=Emulator
+Terminal=true' > ~/.local/share/applications/retropie.desktop
```
Comment (fabianmendes) [inline apps/Retropie/install:34]: =false

**Comment by Botspot** [comment]
> I think this is not correct, or it actually is?

It's not correct. You're right.

**Comment by mobilegmYT** [comment]
🤷‍♂️ I found this from TwisterOS, so it should work on rpi os. Might be wrong though.

**Comment by Botspot** [comment]
The syntax used for echoing was not correct. In any case, this PR is being closed because somebody will get 5 dollars for improving the RetroPie app.

**Pull request closed without merging**